src/lakefs_spec/client_helpers.py

```diff
@@ -190,10 +190,27 @@ def create_tag(
         return client.tags_api.create_tag(repository=repository, tag_creation=tag_creation)
     except ApiException as e:
         if e.status == 409 and exist_ok:
+            logger.warning(f"The tag, '{tag}' already exists. It was not reassigned.")
```

Comment:
Make it short and simple, like `f"tag {tag} already exists and is immutable"`, and then match logs against `"tag .* already exists`.